maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PerformReleaseMojo.java

```diff
@@ -179,16 +139,6 @@ public void execute()
                 releaseDescriptor.setScmSourceUrl( connectionUrl );
             }
 
-            if ( username != null )
-            {
-                releaseDescriptor.setScmUsername( username );
-            }
-
-            if ( password != null )
-            {
-                releaseDescriptor.setScmPassword( password );
-            }
-
```

Comment:
Sure this should be removed?
There is no scenario when during the perform release the username/password are needed?
If there is no such scenario: Why do you keep the scm source url?